README.md

```diff
@@ -13,22 +14,27 @@ module "ctags" {
   environment = var.environment
   product     = var.product
   builtFrom   = var.builtFrom
-  }
+}
 ```
 
-This example shows you how to add  `expiresAfter` tag with a specific expiry date. `expiresAfter` tag only applies to sbox environment. 
+### `expiresAfter`
+
+In the Sandbox environment resources must be tagged with an end date on when they are needed till.
```

Comment:
Maybe something like the below? This sentence doesn't read right to me.
`In the Sandbox environment resources must be tagged with an end date after which they are no longer needed. Resources whose expiresAfter date has passed may/will be deleted`